src/tests/unit/handler.test.js

```diff
@@ -37,11 +74,11 @@ describe("Handler verifies route /journal; path '/journal', httpMethod.GET", ()
   ['/journal', '/publisher'].map(calledPath => (
```

Comment:
Does this test do anything anymore? The commented code implies not — what are the body expectations?

Reply:
The test still verifiest that the routes is recognized and returns 200 OK for those